geopandas/geodataframe.py

```diff
@@ -443,7 +443,7 @@ def plot(self, *args, **kwargs):
 
     def dissolve(self, by=None, aggfunc='first'):
         """
-        Dissolve geometries within `groupby` into single observation. 
+        Dissolve geometries within `groupby` into single observation.
```

Comment:
Can you add a little bit more explanation (eg add a paragraph after leaving a blank line after this first line), maybe with referencing that geometries are combined using `unary_union`?